examples/arithmetic/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MPL-2.0"
 publish = false
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "lib"]
```

Comment:
I'll go ahead and remove this since `arithmetic` doesn't have an xpcshell test, but we do need to do this in every crate we plan to vendor in m-c—since we're importing it as a Rust library via `extern crate`, not a `.dylib`.